server/tests/src/main/scala/sttp/tapir/server/tests/ServerWebSocketTests.scala

```diff
@@ -18,7 +18,8 @@ import sttp.ws.{WebSocket, WebSocketFrame}
 
 abstract class ServerWebSocketTests[F[_], S <: Streams[S], ROUTE](
     createServerTest: CreateServerTest[F, S with WebSockets, ROUTE],
-    val streams: S
+    val streams: S,
+    val concatenateFragmentedFrames: Boolean = false
```

Comment:
I think I'd turn all tests on by default, and only explicitly disable where needed